components/Api/JsonLink/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { FormattedMessage } from 'react-intl';
+import { FaRobot } from 'react-icons/fa';
+import { IconContext } from 'react-icons';
+import styles from './index.module.scss';
+import type { FC } from 'react';
+
+type JsonLinkProps = {
+  fileName: string;
+  version: string;
+};
+
+const JsonLink: FC<JsonLinkProps> = ({ fileName, version }) => (
+  <div className={styles.json}>
+    <a
+      href={`https://nodejs.org/docs/latest-${version}.x/api/${fileName}.json`}
+    >
+      <FormattedMessage id="components.api.jsonLink.title" tagName="span" />
+      <IconContext.Provider value={{ className: styles.FaRobotIcon }}>
```

Comment:
Question, why do we need this IconContext Provider here?

Reply:
As I was saying to Augustin - I used the suggested configuration here https://github.com/react-icons/react-icons#configuration but if we usually pass the styles directly to the icon, happy to switch to that.

Reply:
I have the feeling this Context Provider should probably not be defined on each individual component. Ill give a read to these docs later just to double check

Reply:
I removed it - just in case 🙂 (but could be brought back, although probably not likely to need to do that) Thanks once more!